core/mobile-menu.ts

```diff
@@ -0,0 +1,47 @@
+import { atom, computed } from 'nanostores'
+
+import { fastCategories } from './fast.ts'
+import { router } from './router.ts'
+import { slowCategories } from './slow.ts'
+
+const isMenuShowed = atom<boolean>(false)
```

Comment:
The names are a littler confusing. I can’t understand why do we need this store.

Reply:
I agree, I thought about how to name it and left it like this for now. The idea is that we use this store as a switch for opening and closing the menu, but we also use a filter so as not to open the menu on `/fast` and `/slow` routes if the menu does not have more than 1 category. Maybe name this store `isMenuOpened`, and the combined one `isMenuDisplayed`?

```js

export const isMenuDisplayed = computed(
  [$isMenuOpened, $isMenuHasItems],
  (isOpened, isHasItems) => {
    return isOpened && isHasItems
  }
)

```

Reply:
`Displayed` is a little better.

But maybe we can check `fast` categories count in `openMenu`? I think it will be simpler and less confusing.

Reply:
I need to think about this, because I get some problems if I don't use the store. 

Also I suggest converting `/slow` and `/fast` links into buttons, dynamically removing the link from the `href` attribute, because if the user has a list of categories and wants to open the menu, he always goes to the `general` category. For example:
```js
href={$router.route === 'slow' ? "" : getURL('slow')}
```

Reply:
We need to be able to open link in new tab, this is why we can't convert it to button.